src/book/kitchen-sink/05-images.mdx

```diff
@@ -40,6 +40,41 @@ hidden: true
 
 </Grid>
 
+### Different grid gaps
+
+#### `gap="medium"`
+
+<Grid columns='1fr 1fr' rows="1fr 1fr" gap="medium" caption="Some possible pets">
+
+![Bird](https://source.unsplash.com/featured/960x540/?bird,1 'Bird')
```

Comment:
```suggestion
![Bird](https://source.unsplash.com/featured/960x540/?bird 'Bird')
```

I think that these URLs are incorrect - it seems if unsplash can't find what is specified it will return a random featured photo, but without the `,1` in the URL it returns an accurate hit consistently

![image](https://user-images.githubusercontent.com/80299053/144763408-3388f8ce-fa55-477d-8dbc-ff7026f19291.png)

Reply:
Cat is the supreme animal, clearly

Reply:
Makes sense @LachlanCourt, but we should probably fix this in another PR.